scripts/lib/installer-win32.tsx

```diff
@@ -93,6 +93,7 @@ export default async function buildInstaller(workDir: string, appDir: string, de
     '-sice:ICE61',
     `-dappDir=${ appDir }`,
     '-ext', 'WixUIExtension',
+    '-ext', 'WixUtilExtension',
     '-nologo',
     '-out', path.join(process.cwd(), 'dist', `Rancher Desktop Setup ${ appVersion }.msi`),
```

Comment:
There should be a line announcing that the MSI was successfully built in this path after `light.exe` finishes successfully.

Reply:
Added in 197372ea93ae0559f39da11dcdc16447413f1e8a.